etlhelper/db_params.py

```diff
@@ -29,6 +29,12 @@ def __getattr__(self, item):
             # https://docs.python.org/3/library/functions.html#getattr
             raise AttributeError(f'No such attribute: {item}')
 
+    def __setattr__(self, item, value):
+        if item not in self.__dir__():
+            raise AttributeError(f"{item} is not a valid DbParams attribute")
+        # TODO: Check for string value?
+        super().__setattr__(item, value)
```

Comment:
Why `super`? `__getattr__` uses `self`.

Reply:
I've pushed a commit with change; see what you think.  I prefer the dictionary-style `self[item] = value` because I'm a bit scared of `super` and still think of it as a bit magic.